docs/developers-clear-containers-install.md

```diff
@@ -168,3 +168,5 @@ $ make help
 
   * [General Debugging](../README.md#debugging)
   * [Debugging the agent inside the hypervisor](debug-agent.md)
+  * [Debugging the kernel inside the hypervisor](https://github.com/mcastelino/runtime/blob/topic/docs/docs/debug-kernel.md)
```

Comment:
Could you update the link to be the one in `clearcontainers` repo not your private repo @mcastelino

Reply:
Oh I just recall that this is udpated automatically with the merge..

Reply:
@gorozco1 - I think your original comment is correct - the doc already exists in the runtime repo so the doc needs to reference:

```
https://github.com/clearcontainers/runtime/blob/master/docs/debug-kernel.md
```

Reply:
Once the link is updated to a clearcontainers repo I am happy to approve.